hippod/snippet_db.py

```diff
@@ -12,14 +13,25 @@
 log = logging.getLogger()
 
 
-def exchange_entry(app, sha, data, source_path, source_type):
+def get_size(snippet_db_path):
+    image_path = os.path.join(snippet_db_path)
+    with open(image_path, 'rb') as f:
+        content = f.read()
+        data = base64.b64encode(content)
+        bin_data = base64.b64decode(data)
+        size_real = len(bin_data)
+    return size_real
+
+
+def exchange_entry(app, sha, data, source_path, source_type, snippet_db_path):
     path = os.path.join(source_path)
     f_format = data['mime-type'].split('-')[-1]
     if 'name' not in data or data['name'] is None:
         entry = dict()
         entry['data-id'] = sha
         entry['type'] = 'snippet'
         entry['name'] = 'snippet-image-{}.{}'.format(sha, f_format)
+        entry['size-real'] = get_size(snippet_db_path)
```

Comment:
why, I mean before I also displayed the real-size, why doit again. Why is this not stored in the meta-data at the time a entry is received?